packages/grid/scss/_mixins.scss

```diff
@@ -320,9 +327,12 @@ $carbon--aspect-ratios: ((16, 9), (2, 1), (4, 3), (1, 1), (1, 2));
     @include carbon--make-row();
   }
 
-  .#{$prefix}--grid--condensed [class*='#{$prefix}--col'] {
-    padding-top: $condensed-gutter / 2;
-    padding-bottom: $condensed-gutter / 2;
+  .#{$prefix}--grid--condensed .#{$prefix}--row:not(:last-of-type) {
```

Comment:
I think we'll want to keep the padding changes from: https://github.com/carbon-design-system/carbon/pull/4960 👍

Reply:
Not sure how that happened, I must have merged something incorrectly. Git can be confusing haha, a778420

Reply:
Totally get that 😂 😂